src/components/GroupSystems/GroupSystems.cy.js

```diff
@@ -309,7 +310,7 @@ describe('selection and bulk selection', () => {
   });
 
   it('can select and deselect systems', () => {
-    const middleRow = Math.ceil(DEFAULT_ROW_COUNT / 4);
+    const middleRow = Math.ceil(DEFAULT_ROW_COUNT / 7.5);
```

Comment:
![red-sus-among-us_bGhsa2aUmZqaraWkpJRnZWltrWdlaW0-3351645577](https://github.com/RedHatInsights/insights-inventory-frontend/assets/7757/74a10a95-954f-4dbc-8902-52a6951c530e)

Reply:
lol cypress was giving an error where if it couldnt actually see the item being selected on the viewport, it would fail. So i increased the value so the item selected would be higher up/cypress would always see it